scss/_modal.scss

```diff
@@ -147,8 +152,10 @@
   flex-shrink: 0;
   align-items: center; // vertically center
   justify-content: flex-end; // Right align buttons with flex property because text-align doesn't work on flex items
-  padding: $modal-inner-padding - $modal-footer-margin-between / 2;
-  border-top: $modal-footer-border-width solid $modal-footer-border-color;
+  // padding: $modal-inner-padding - $modal-footer-margin-between / 2;
+  padding: $modal-footer-padding-top $modal-footer-padding-sides;
```

Comment:
Bootstrap has `$modal-header-padding-y`, `$modal-header-padding-x`, and `$modal-header-padding`.

We should try to maintain a consistent naming convention. I think creating `$modal-footer-padding` might be enough, since x and y for header and footer should be the same (maybe?).